sql/core/src/test/resources/sql-tests/inputs/random.sql

```diff
@@ -22,21 +22,44 @@ SELECT uniform(0, 1, 0) AS result;
 SELECT uniform(0, 10, 0) AS result;
 SELECT uniform(0L, 10L, 0) AS result;
 SELECT uniform(0, 10L, 0) AS result;
+SELECT uniform(0, cast(10 as tinyint), 0) AS result;
+SELECT uniform(0, cast(10 as bigint), 0) AS result;
```

Comment:
This test is the same as `SELECT uniform(0, 10L, 0) AS result;`. Shall we test a different type like `smallint`?

Reply:
Good idea, updated.